src/material-examples/sidenav-backdrop/sidenav-backdrop-example.css

```diff
@@ -4,3 +4,10 @@
   margin: 10px;
   border: 1px solid #555;
 }
+
+mat-drawer-content {
+  padding: 10px;
```

Comment:
A little nitpicky, but if I remember correctly, all the spacings in the Material design spec were in 4px increments. Maybe we should switch all of these 10px values to 8px or 12px?

Reply:
@crisbeto I switched all values to either 12px or 8px. Thanks